x/blog/handler_test.go

```diff
@@ -880,12 +893,21 @@ func TestCreateArticle(t *testing.T) {
 				}
 
 				// ensure createdAt is after test execution starting time
-				createdAt := stored.CreatedAt
-				weave.InTheFuture(ctx, createdAt.Time())
 
-				// avoid registered at missing error
-				tc.expected.CreatedAt = createdAt
+				assert.Equal(t, false, weave.InTheFuture(ctx, stored.CreatedAt.Time()))
+
+				// avoid createdAt missing error
+				tc.expected.CreatedAt = stored.CreatedAt
+				// avoid deleteTaskID missing error
+				tc.expected.DeleteTaskID = stored.DeleteTaskID
 				assert.Equal(t, tc.expected, &stored)
+
+				// test if deletion task is scheduled
+				if stored.DeleteTaskID != nil {
+					if err := scheduler.Delete(kv, stored.DeleteTaskID); err != nil {
```

Comment:
I'm getting an error here. Apparently scheduled needs to be injected to a BaseApp to work but in the test suite, I could find no way to initiate a mock app. Any help?

Reply:
This is a bug in `weavetest.Cron` implementation. Fixed in https://github.com/iov-one/weave/pull/1012